src/openrct2/ride/TrackPaint.h

```diff
@@ -362,14 +362,113 @@ constexpr uint32_t kTrackSpritesSubmarineRideMiniHelicoptersQuarterTurn1Tile[4]
     SPR_TRACK_SUBMARINE_RIDE_MINI_HELICOPTERS_FLAT_QUARTER_TURN_1_TILE_SE_SW,
 };
 
-extern const uint8_t mapLeftQuarterTurn5TilesToRightQuarterTurn5Tiles[];
-extern const Direction mapReversedDiagonalStraight[];
+constexpr uint8_t kMapLeftQuarterTurn5TilesToRightQuarterTurn5Tiles[] = {
+    6, 4, 5, 3, 1, 2, 0,
+};
+constexpr uint8_t kMapReversedDiagonalStraight[4] = {
+    3,
+    2,
+    1,
+    0,
+};
 
-extern const CoordsXY defaultRightQuarterTurn5TilesOffsets[4][5];
-extern const CoordsXYZ defaultRightQuarterTurn5TilesBoundOffsets[4][5];
-extern const CoordsXY defaultRightQuarterTurn5TilesBoundLengths[4][5];
+constexpr CoordsXY kDefaultRightQuarterTurn5TilesOffsets[4][5] = {
```

Comment:
I think these shouldn't be directly in the header, as this would create a new instance every time the header is included. Rather, they should remain in TrackPaint.cpp and exposed through a getter function.

Reply:
Should I do the same for the others that I did in https://github.com/OpenRCT2/OpenRCT2/pull/21942 etc?

Reply:
I think that @Harry-Hopkinson did this because Matt requested making the segment declarations `constexpr`. I can imagine this will be confusing, as it now looks like two team members are asking opposite things.

Let’s ask @ZehMatt to weigh in.

Reply:
When to use getters instead of constexpr is situational. I think in this case it might be best to move them into a new header and have only cpp files include it, adding getters always adds the risk of adding overhead and I think this is one of those areas in the code where avoiding branching code is probably for the better.

Reply:
So do you mean they should stay as they are now (extern), or?

Reply:
I would go for a new header.

Reply:
A new header, and changing them to constexpr, you mean?

If many cpp files pull from that header, doesn’t that create copies? Or will the compiler handle it correctly?

Reply:
Yes, that would create new instances of the arrays. But it also allows the compiler to inline them, potentially unroll a few loops, etc. As Matt said, it's situational 😅

Reply:
I'm a bit puzzled to why this PR changes them to begin with, best to revert that and just stick to refactoring the names. This is beyond a style refactor when start moving data structures around.

Reply:
Fair enough. I’ll probably touch them anyway in a future track refactor.

@Harry-Hopkinson Could you take out the constexpr changes and just leave in the name changes?